modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseInit.java

```diff
@@ -0,0 +1,40 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+
+public class ClickHouseInit {
```

Comment:
In this class, we should only make the constants public that need to be public and keep the rest package-protected. The class itself can also be package-protected, or?

Reply:
Main reason this class isn't package-protected is that I was trying to follow the package layout you picked which moves the junit-tests into a dedicated folder.
From there I can't access the class to get the constants for testing.
If you are ok with merging the packages I can male it protected.

Reply:
I think https://github.com/testcontainers/testcontainers-java/pull/4038#issuecomment-927733013 will make it obsolete